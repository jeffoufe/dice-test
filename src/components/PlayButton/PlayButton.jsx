// @flow

import React from 'react';
import styles from './PlayButton.module.scss';

const PlayButton = () => (
    <div className={styles['play-container']}>
        <div className={styles['play-button']}>&#9658;</div>
    </div>
)

export default PlayButton;