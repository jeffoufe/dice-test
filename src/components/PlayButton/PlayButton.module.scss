.container {
    position: absolute;
    width: 50px;
    height: 50px;
    left: 0;
    bottom: 4px;
    color: white;
    background-color: rgba(0, 0, 0, 0.5);
    text-align: center;
    cursor: pointer;
}

.play-button {
    display: inline-block;
    margin-top: 50%;
    transform: translateY(-50%);
    font-size: 25px;
}