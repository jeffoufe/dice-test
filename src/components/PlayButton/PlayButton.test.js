import PlayButton from './PlayButton';
import { shallow } from 'enzyme';
import React from 'react';

describe('Featured', () => {
    const wrapper = shallow(
        <PlayButton />
    );

    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    })
})