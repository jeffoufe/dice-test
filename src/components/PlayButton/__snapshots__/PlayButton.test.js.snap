// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Featured renders correctly 1`] = `
<div
  className="play-container"
>
  <div
    className="play-button"
  >
    â–º
  </div>
</div>
`;
